src/lumigo_tracer/utils.py

```diff
@@ -67,6 +75,16 @@ def config(
     Configuration.is_step_function = step_function
     Configuration.timeout_timer = timeout_timer
     Configuration.send_only_if_error = os.environ.get("SEND_ONLY_IF_ERROR", "").lower() == "true"
+    if "LUMIGO_DOMAINS_SCRUBBER" in os.environ:
+        try:
+            Configuration.domains_scrubber = json.loads(os.environ["LUMIGO_DOMAIN_SCRUBBER"])
+        except Exception:
+            get_logger().critical(
+                "Could not parse the specified domains scrubber, shutting down the reporter."
+            )
+            Configuration.should_report = False
+    else:
```

Comment:
Shouldn't it be the opposite in the if-else? Don't we prefer that the argument will override the environment variable?

Reply:
I don't think so 😮 
I think that the more flexible parameter should override less flexible, and environment variable is much more flexible than code 😮

Reply:
you won